bindings/Python/py11File.cpp

```diff
@@ -231,12 +235,13 @@ pybind11::array File::Read(const std::string &name, const Dims &start,
         return pyArray;
     }
 
-    return Read(name, start, count, 0, 0, blockID);
+    return Read(name, start, count, 0, 0, blockID, order);
 }
 
 pybind11::array File::Read(const std::string &name, const Dims &start,
                            const Dims &count, const size_t stepStart,
-                           const size_t stepCount, const size_t blockID)
+                           const size_t stepCount, const size_t blockID,
+                           const std::string &order)
```

Comment:
I'd use `const std::string order` as that's what is used for strings we know are always small.